src/hooks/useV2Pairs.ts

```diff
@@ -22,91 +21,103 @@ export enum PairState {
 }
 
 export function useV2Pairs(currencies: [Currency | undefined, Currency | undefined][]): [PairState, Pair | null][] {
-  const [reserves, setReserves] = useState<
-    ({ reserve0: string; reserve1: string; poolId: number; totalSupply: string } | number | undefined)[]
-  >([]);
-
   const tokens = useMemo(() => {
     return currencies.map(([currencyA, currencyB]) => [currencyA?.wrapped, currencyB?.wrapped]);
   }, [currencies]);
 
-  // const lastBlockNumber = useBlockNumber();
+  const [pairs, setPairs] = useState<[PairState, Pair | null][]>(Array(tokens.length).fill([PairState.LOADING, null]));
+
+  const last = useLastCount(20000);
```

Comment:
any reason to set 20s period?

Reply:
The number of API requests was still large when set to 10s. Let's test this more and set a reasonable period.